lib/renditions/utils.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from 'path';
+import { Component, ComponentRendition, ComponentsDefinition, GetFileContentType } from '../models';
+import merge = require('lodash.merge');
+import { deepFreeze } from '../util/freeze';
+
+export async function loadRenditions(
+    componentsDefinition: ComponentsDefinition,
+    getFileContent: GetFileContentType,
+): Promise<ComponentsDefinition> {
+    const enrichedDefinition = merge({}, componentsDefinition);
```

Comment:
Just for my understanding, this 'unfreezes' an object?

Reply:
It creates a [copy of the object](https://lodash.com/docs#merge)

Reply:
Shouldn't you use lodash deepClone in this case? Or is it different?

Reply:
The result is the same. I copied this from the old code, but I guess that for new code we're better off using the function that describes best what it does: cloneDeep.